src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -295,21 +295,26 @@ private static PackageReference ToPackageReference(LibraryDependency library, Nu
                 .GetPackageReferencesAsync(targetFramework, CancellationToken.None))
                 .ToList();
 
-            var packageTargetFallback = _vsProjectAdapter.PackageTargetFallback?.Split(new[] { ';' })
+            var splitChars = new[] { ';' };
+
+            var packageTargetFallback = _vsProjectAdapter.PackageTargetFallback?.Split(splitChars)
                 .Select(NuGetFramework.Parse)
-                .ToList();
+                .ToList()
```

Comment:
Although it wasn't covered there before, what happens in case of empty or invalid entries in the list?